app/views/instruments/schedule.html.haml

```diff
@@ -25,25 +25,31 @@
     class: "btn-add"
 
   - if current_ability.can?(:create, OfflineReservation)
-    - if @product.online?
+    - if @product.offline?
+      = link_to t(".bring_instrument_online"),
+        facility_instrument_bring_online_path(current_facility, @product),
+        method: :put, class: "btn-add"
+    - elsif @product.partially_available?
+      = link_to t(".mark_instrument_fully_available"),
+        facility_instrument_partial_availability_path(current_facility, @product),
+        method: :delete, class: "btn-add"
+    - else
       = link_to t(".mark_instrument_offline"),
         new_facility_instrument_offline_reservation_path(current_facility, @product),
         class: "btn-add"
-
-    - else
-      = link_to t(".bring_instrument_online"),
-        facility_instrument_bring_online_path(current_facility, @product),
-        method: :put,
+      = link_to t(".mark_instrument_partially_available"),
+        new_facility_instrument_partial_availability_path(current_facility, @product),
         class: "btn-add"
 
   = render_view_hook "after_offline_toggle"
 
 - if @admin_reservations.empty?
   %p.notice= t(".no_upcoming_admin_reservations")
-
 - else
   - if @product.offline?
     %p.alert.alert-danger= t(".instrument_is_offline")
+  - elsif @product.partially_available?
```

Comment:
This gets hidden when there are no upcoming reservations. I feel like we should put that out of the surrounding if/else so it and the offline banners are always visible.

<img width="912" alt="screen shot 2018-11-05 at 12 27 04 pm" src="https://user-images.githubusercontent.com/1099111/48018349-3c1e5e80-e0f6-11e8-972b-210564c3c43f.png">

Reply:
The trick here is that if there’s an offline reservation, that is an admin hold, so it would appear. I’ll move out the instrument alert notice, though!